app/Models/Country.php

```diff
@@ -10,4 +10,11 @@ class Country extends Model
     use HasFactory;
 
     protected $fillable = ['name', 'short_code'];
+
+    protected $hidden = ['created_at', 'updated_at'];
```

Comment:
@jstarnate I don't really understand the purpose of `$hidden` here, can you explain?

Reply:
@PovilasKorop I watched one of your YT vids about excluding unnecessary attributes from the API, so since the results don't care about timestamps, I decided to hide those two attributes :)

Reply:
Makes sense but we don't have API here :)